### PR TITLE
Production

### PR DESCRIPTION
**Pull request by dirkkelly**

### CODE REVIEW

File: app/models.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 require 'sequel'
 
 ENV['DATABASE_URL'] ||= "postgres://localhost/citygram_#{Citygram::App.environment}"
-DB = Sequel.connect(ENV['DATABASE_URL'])
+DB = Sequel.connect(ENV['DATABASE_URL'], max_connections: 20)
```
Comment (dirkkelly) [inline app/models.rb]: @fma2 I believe this will fix the timeout issues. I've tested it with staging emails, now we need to test with live.

**Comment by fma2** [comment]
Ok, trying to catch up on all of your work. Good find with that [doc](https://github.com/ollie/connection-pool-concurrency/blob/master/README.md)!  Still wondering - why were the connections not a noticeable (might have been failing but we didn't know it) issue until now?  

Looking at what you documented in [issue #57 ](https://github.com/BetaNYC/citygram-nyc/issues/57) and checking with our Heroku resources.  Is this how you reached 20 databased connections: `1 dyno * 1 worker * 20 threads = 20 connections (pool is 20)`?

**Comment by dirkkelly** [comment]
@fma2 yeah I essentially guessed after reading that document. I want to go over it closer with @excepttheweasel on wednesday. 

I think the reason for failure was the change to the new database instances, this was done about 3 weeks ago

**Comment by dirkkelly** [comment]
Merged because it's worked so reliably last few weeks.

I'm not sure I got the calculation right on this. @excepttheweasel does unicorn have threads?

**Comment by excepttheweasel** [comment]
@dirkkelly Unicorn actually uses `fork`, not threads to achieve concurrency. Not sure I see the relevance here?

**Comment by dirkkelly** [comment]
Roger, yeah I was reading the wrong instructions (puma instead of unicorn) when calculating the max connections.

We have `1 dyno * 2 workers * 1 thread = 20 connections`

Same answer, different calculation

**Pull request merged**